dashdotdb/schemas/swagger.yaml

```diff
@@ -11,38 +11,55 @@ info:
 servers:
 - url: /api/v1
 paths:
-  /imagemanifestvuln/metrics:
+  # Tokens
+  /token:
     get:
-      operationId: dashdotdb.controllers.metrics.imagemanifestvuln.search
-      summary: Lists Image Manifests Vulnerabilities
+      summary: Create a token for data upload
+      parameters:
+      - name: scope
+        in: query
+        required: true
+        schema:
+          type: string
       responses:
         "200":
-          description: successful operation
+          description: token created
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ImageManifestVulnReport'
-                x-content-type: application/json
-  /deploymentvalidation/metrics:
-    get:
-      operationId: dashdotdb.controllers.metrics.deploymentvalidation.search
-      summary: Lists Deployment Validation Information
+                type: object
+                properties:
+                  token:
+                    type: string
+                    description: A unique token identifier
+        "405":
+          description: invalid scope
+    post:
+      summary: Close a token so it can not receive data
+      parameters:
+      - name: token
+        in: query
+        required: true
+        schema:
+          type: string
+      - name: scope
```

Comment:
you don't need that with @apahim  propsal

Reply:
the only way that a client knows about a token is because they acquired it... anyway, if you feel strongly about this, let's leave it